environments/elk-ror/docker-compose.yml

```diff
@@ -22,6 +22,9 @@ services:
       - bootstrap.memory_lock=true
       # for a sake of debugging and profiling
       - "ES_JAVA_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5000 -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=9010 -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.rmi.port=9010 -Djava.rmi.server.hostname=127.0.0.1"
+      - KIBANA_SERVICE_ACCOUNT_TOKEN=UNUSED
```

Comment:
Must we pass these dummy ECK-based values?

Reply:
For now - yes. ROR settings are a little bit different in the case of ECK (we have to add two more blocks: for the probe and for a sake of the ECK operator... and we cannot use /at least in a simple way/ kibana user:pass block).
These ENVs are used in these additional blocks. Currently, ROR ES doesn't support fallback to default value. When ENV is not found, ROR settings fail to reload. That's why we have to add the dummy values here.